### PR TITLE
[CIVIS-9031] Deprecate `metrics` cmd in favour of `measures` cmd

### PR DESCRIPTION
**Pull request by ManuelPalenzuelaDD**
This patch renames the `metrics` command to `measures`. We'll still support the `metrics` command name but it'll print a deprecation warning when used. Additionally, we do the same change to the `--metrics` option of the command (renaming it to `--measures`)

Context: https://docs.google.com/document/d/1X9-0eiiNaDsfLMKlVYsGdQs4RDKGZx9jEqLX4NEPAxY

This is a sister pr of https://github.com/DataDog/datadog-ci/pull/1203

- [x] Feature or bugfix MUST have appropriate tests (unit, integration)

### CODE REVIEW

File: src/commands/measure/README.md [hunk 1]
```diff
@@ -11,14 +11,14 @@ datadog-ci metric [--no-fail] [--level <pipeline|job>] [--metrics]
 For example:
 
 ```bash
-datadog-ci metric --level job --metrics binary.size:1024
+datadog-ci measure --level job --metrics binary.size:1024
```
Comment (juan-fernandez) [inline src/commands/measure/README.md:14]:
```suggestion
datadog-ci measure --level job --measures binary.size:1024
```

File: README.md [hunk 2]
```diff
@@ -66,9 +66,9 @@ See each command's linked README for more details, or click on [📚](https://do
 - `instrument`: Apply Datadog instrumentation to the given [AWS Lambda functions](src/commands/lambda).
 - `uninstrument`: Revert Datadog instrumentation from the given [AWS Lambda functions](src/commands/lambda).
 
-#### `metric`
+#### `measure`
 
-- Add [metrics](src/commands/metric) to a CI Visibility Pipeline trace or Job span in Datadog. [📚](https://docs.datadoghq.com/continuous_integration/pipelines/custom_tags_and_metrics/)
+- Add [measures](src/commands/measure) to a CI Visibility Pipeline trace or Job span in Datadog. [📚](https://docs.datadoghq.com/continuous_integration/pipelines/custom_tags_and_metrics/)
```
Comment (juan-fernandez) [inline README.md:71]: is the idea to update the documentation too? We refer to metrics still. We might want to reflect that this link is not updated somewhere

**Review by juan-fernandez — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (ManuelPalenzuelaDD) [reply README.md:71]: Yes, I'm currently changing the docs too (https://github.com/DataDog/documentation/pull/21897).
I will rename the mentions to the outdated docs as soon as that is merged.

File: src/commands/measure/README.md (hunk 1 above)
Comment (ManuelPalenzuelaDD) [reply src/commands/measure/README.md]: Nice catch, thank you!

File: src/commands/measure/README.md [hunk 3]
```diff
@@ -40,19 +40,19 @@ export DD_API_KEY='<API key>'
...
 ```
 
 ## Further reading
 
 Additional helpful documentation, links, and articles:
 
-- [Learn about Adding Custom Tags and Metrics to Pipeline Traces][1]
+- [Learn about Adding Custom Tags and Measures to Pipeline Traces][1]
 
 [1]: https://docs.datadoghq.com/continuous_integration/pipelines/custom_tags_and_metrics/
```
Comment (aliciascott) [inline src/commands/measure/README.md:58]:
```suggestion
[1]: https://docs.datadoghq.com/continuous_integration/pipelines/custom_tags_and_metrics/
```

Just noting this link will need changed to the new page

**Review by aliciascott — APPROVED** [review]
Approved; pending changing the link to the newly named page as well.

**Pull request merged**